manifests/i/isnetwork/ISNAutoItStudio/1.13/isnetwork.ISNAutoItStudio.locale.de-DE.yaml

```
@@ -0,0 +1,19 @@
+# Created using wingetcreate 0.3.0.3
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.0.0.schema.json
+
+PackageIdentifier: isnetwork.ISNAutoItStudio
+PackageVersion: 1.13
+PackageLocale: de-DE
+Publisher: isnetwork
+PublisherUrl: https://www.isnetwork.at
+PublisherSupportUrl: https://www.isnetwork.at

```

Publisher needs to match the value in `A`dd or `R`emove `P`rograms

```suggestion
Publisher: ISI360
PublisherUrl: https://www.isnetwork.at
PublisherSupportUrl: https://www.isnetwork.at/isn-support/
```